CefSharp.Wpf/Experimental/WpfIMEKeyboardHandler.cs

```diff
@@ -40,6 +40,25 @@ public WpfImeKeyboardHandler(ChromiumWebBrowser owner) : base(owner)
         {
         }
 
+        /// <summary>
+        /// Get the outermost element of the browser 
+        /// </summary>
+        /// <param name="control">The browser</param>
+        /// <returns>The outermost element </returns>
+        FrameworkElement GetOutermostElement(FrameworkElement control)
```

Comment:
Please make this method private and move it to the bottom of the class with the other private methods.

Reply:
ok.